bittide-instances/src/Bittide/Instances/Tests/FullMeshHwCc.hs

```diff
@@ -86,7 +95,8 @@ goFullMeshHwCcTest refClk sysClk rst rxns rxps miso =
   , stats
   , spiDone
   , spiOut
-  , allStable1
+  , transceiversFailedAfterUp
+  , fincFdecIla `hwSeqX` allStable1
```

Comment:
It's looks kind of a random choice to attach the ILA here. I would prefer attaching it to the whole tuple to make it clear that it's part of `goFullMeshHwCcTest` and not particularly related to `allStable1`.

Reply:
Agreed, placed it next to the full tuple.